qt/python/mantidqt/widgets/fitpropertybrowser/fitpropertybrowser.py

```diff
@@ -109,16 +109,21 @@ def _get_selected_workspace_artist(self):
             if ws_artists.workspace_index == self.workspaceIndex():
                 return ws_artists
 
-    def _set_normalise_data_from_workspace_artist(self):
+    def _set_normalise_data(self):
         """
         Set if the data should be normalised before fitting using the
         normalisation state of the active workspace artist.
-        """
-        if AnalysisDataService.doesExist(self.workspaceName()) \
-                and isinstance(AnalysisDataService.retrieve(self.workspaceName()), ITableWorkspace):
-            return
+        If the workspace is a distribution normalisation is set to False so it is not normalised twice.
+        """
+        ws_is_distribution = False
+        if AnalysisDataService.doesExist(self.workspaceName()):
+            workspace = AnalysisDataService.retrieve(self.workspaceName())
+            if isinstance(workspace, ITableWorkspace):
+                return
+            if hasattr(workspace, 'isDistribution') and workspace.isDistribution():
+                ws_is_distribution = True
         ws_artist = self._get_selected_workspace_artist()
-        self.normaliseData(ws_artist.is_normalized)
+        self.normaliseData(ws_artist.is_normalized and not ws_is_distribution)
```

Comment:
This is a nitpick but I just had to dig to find out what this flag does

Could we change this to
```python
should_normalise_before_fit = ws_artist.is_normalized and not ws_is_distribution
self.normaliseData(should_normalise_before_fit)
```